indico/web/forms/fields.py

```diff
@@ -249,6 +249,22 @@ def __init__(self, *args, **kwargs):
         super(IndicoPasswordField, self).__init__(*args, **kwargs)
 
 
+class CategoryField(HiddenField):
+    """Field that lets you select a category"""
+
+    widget = JinjaWidget('forms/category_picker_widget.html')
+
+    def process_formdata(self, valuelist):
+        from indico.modules.categories.models.categories import Category
```

Comment:
Eventually (2.0) I'd like that (i.e. import all the models from a module in its `__init__.py`); right now it'd be problematic in some cases due to circular import (possibly related to top-level imports from/of legacy code). But in cases where we already have it in `__init__.py` I don't see a reason against using it